frontend/www/js/course.list.js

```diff
@@ -13,7 +13,7 @@ omegaup.OmegaUp.on('ready', function() {
       course.submissionsListUrl = '/course/' + course.alias + '/list/';
       course.submissionsList = omegaup.T.courseListSubmissionsByGroup;
       course.activity = omegaup.T.wordsActivityReport;
-      if (course.finish_time > Date.now()) {
+      if (course.unix_finish_time > Date.now()) {
```

Comment:
si el problema es que está como fecha, la solución es tratarla como tal:

```suggestion
      if (course.finish_time.getTime() > Date.now()) {
```

Reply:
Listo